src/include/basil.h

```diff
@@ -565,11 +640,22 @@ typedef struct basil_request_query_status {
 	basil_request_query_status_res_t *reservation;
 } basil_request_query_status_t;
 
+typedef struct basil_topology_filter {
+	char name[BASIL_STRING_LONG];
+	struct basil_topology_filter *next;
+} basil_topology_filter_t;
+
+typedef struct basil_request_query_topology {
+    int executeFilters;
+	basil_topology_filter_t *filters;
```

Comment:
the indentation within this structure looks off.  Please double check, and please use tabs for indentation, not spaces.